packages/config/src/config/build.js

```diff
@@ -41,7 +41,9 @@ export default () => ({
     },
     css: {},
     cssModules: {
-      localIdentName: '[local]_[hash:base64:5]'
+      modules: {
+        localIdentName: '[local]_[hash:base64:5]'
```

Comment:
Do you think is it good to normalize in `options.js` or make a warning about config breaking change?

Reply:
I think it would be better to keep consistency with css-loader, so maybe warning is better

Reply:
css-loader is a very basic loader, v3 breaking changes must have its consideration, so I think it’s better to align its behavior.
And it only affects users who have customized cssModues config, they should have a clear idea about what need to be changed

Reply:
BTW, css-loader has options schema validation 😺